### PR TITLE
ability to navigate away from survey to homepage

### PR DESCRIPTION
**Pull request by fallguy**
* Ability to navigate from survey page to home page
* adds survey object outside of notification object as well as inside
* adds user_id property outside of notification
* adds location object to user's new submission
* adjusts newSubmission definition to take dynamic objects for survey, notification, and location rather than specifying each prop

### CODE REVIEW

**Comment by fallguy** [comment]
@fchau-arrdude & @vvvli - can one of you review this to make sure it's working as intended?

File: src/SliderWidget.js [hunk 1]
```diff
@@ -11,74 +13,82 @@ var currentUnixTime = Math.round((new Date()).getTime() / 1000);
...
     }
 	}
 
+  componentDidMount() {
+    this._getLocationAsync();
+    let notificationObject = this.state.notification;
+    this.setState({ notificationObject: notificationObject });
+  }
+
   getVal(val) {
```
Comment (vvvli) [inline src/SliderWidget.js:47]: Is this function used or going to be used at all?

**Review by vvvli — APPROVED** [review]
@fallguy I see the location data is passed into the newSubmission object after the user hits submit.

**Review by vvvli — APPROVED** [review]
@fallguy Importing a react library called "react-navigation" to let the user move to a different page in the app. In this case, we move from surveys to the "profile" page.

**Pull request merged**